apps/crunchy-postgres/workload/run_litmus_test.yml

```diff
@@ -0,0 +1,28 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: cruncy-postgres-loadgen 
+  namespace: litmus
+spec:
+  template:
+    metadata:
+      name: crunchy_postgres-loadgen
+      namespace: litmus
+      labels:
+        app: crunchy-postgres-loadgen
```

Comment:
Use the label key as 'loadgen' for load generator pod in all the load generators.

Reply:
@gprasath We cant use same app labels for all loadgens..there may be a scenario where two loadgens running on same cluster...then in that case there will be a contradiction.

Reply:
@gprasath I have made this change in actual loadgen job